### DISCUSSION TITLE
monika cant read my gifts.

### DISCUSSION THREAD

**Discussion by iloveyou12282** [Monika-After-Story/MonikaModDev#9774] [general - spritepacks]
I'm sure everyone knows how to do this now, but monika cant read my gifts. she sees im trying to give her something, but cant read it..help please?

**Comment by Wingdinggaster656** [comment]
What's your gift?

**Reply by iloveyou12282** [reply]
briaryoung-sleeveless turtleneck black

**Reply by Wingdinggaster656** [reply]
Sounds like a spritepack gift, which means you must install the spritepack first. You sure you have installed it already?

**Reply by iloveyou12282** [reply]
yes, i downloaded it two days ago. whats weird is she was able to get the gifts the first 3 times I gave her a gift. now she can't read them.

**Reply by iloveyou12282** [reply]
even though I gave her 3 gifts successfully, is there a chance I installed in wrong somehow?

**Comment by Booplicate** [comment]
[Follow these instructions](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation).

**Reply by iloveyou12282** [reply]
she still cant read them.

**Reply by Booplicate** [reply]
I don't think you followed the instructions, they tell you what to do depending on the step you're stuck at.

**Reply by iloveyou12282** [reply]
ah, alright. thats my bad.
